tests/backend/realm/test_realm_stats.py

```diff
@@ -0,0 +1,39 @@
+# Parsec Cloud (https://parsec.cloud) Copyright (c) AGPLv3 2019 Scille SAS
+
+import pytest
+from uuid import uuid4
+
+from tests.backend.common import realm_create, realm_stats
+from tests.backend.common import vlob_create, block_create
+
+
```

Comment:
Other interesting tests:
- test stats not allowed for user with no access on the realm
- test stats not allowed for user with similar device_id but on a different organization (see https://github.com/Scille/parsec-cloud/blob/b735aede870925e724360594155ce426e41c2796/tests/backend/realm/test_vlob_access.py#L328-L335)
- test stat on unknown realm_id